src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -173,6 +171,11 @@ public FormValidation doCheckReturnStatus(@QueryParameter boolean returnStdout,
             if (returnStdout) {
                 durableTask.captureOutput();
             }
+            if (step.encoding != null) {
+                durableTask.charset(Charset.forName(step.encoding));
```

Comment:
Catch/rethrow exception with more debug details?

Reply:
Which debug details are you looking for here?